lucille-core/src/test/java/com/kmwllc/lucille/stage/ReplacePatternsTest.java

```diff
@@ -38,6 +38,46 @@ public void testReplacePatterns() throws Exception {
     assertEquals("This should be untouched", doc3.getStringList("output3").get(0));
   }
 
+  @Test 
+  public void testIgnoreCase() throws Exception {
+    Stage stage = factory.get("ReplacePatternsTest/one.conf");
+
+    Document doc = Document.create("doc");
+    doc.setField("input1", "The term fAlse should be replaced.");
+    stage.processDocument(doc);
+    assertEquals("The term REPLACED should be replaced.", doc.getStringList("output1").get(0));
+  }
+
+  @Test 
+  public void testMultiline() throws Exception {
+    Stage stage = factory.get("ReplacePatternsTest/two.conf");
+
+    Document doc = Document.create("doc");
+    doc.setField("input1", "The term fAlse should be replaced,\n but not this false");
```

Comment:
This test is focusing on multiline handline not case-insensitivity so I'd suggest changing the first "fAlse" to "false" to simplify/clarify the scenario. Given that there's a separate test for case insensitivity. Please give the conf files descriptive names if possible, not one.conf, two.conf, three.conf etc.